node/src/BaseClient.ts

```diff
@@ -938,6 +941,7 @@ export class BaseClient {
      * See https://valkey.io/commands/getdel/ for details.
      *
      * @param key - The key to retrieve from the database.
+     * @param decoder - Optional enum parameter for decoding the response.
```

Comment:
I think we need a bit better doc + align it with others.
```suggestion
     * @param decoder - (Optional) {@link Decoder} type which defines how to handle the response. If not set, the default decoder from the client config will be used.
```
This will add a clickable link to the doc.

or even
```suggestion
     * @param decoder - (Optional) {@link Decoder} type which defines how to handle the response. If not set, the {@link BaseClientConfiguration.defaultDecoder|default decoder} will be used.
```

Reply:
Please update `get` doc too.